integrationTests/DoctrineDonationRepositoryTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace MatchBot\IntegrationTests;
+
+use Doctrine\ORM\EntityManagerInterface;
+use MatchBot\Domain\DayOfMonth;
+use MatchBot\Domain\DonationRepository;
+use MatchBot\Domain\DonationSequenceNumber;
+use MatchBot\Domain\DonationStatus;
+use MatchBot\Domain\DonorAccount;
+use MatchBot\Domain\DonorName;
+use MatchBot\Domain\EmailAddress;
+use MatchBot\Domain\Money;
+use MatchBot\Domain\PaymentMethodType;
+use MatchBot\Domain\PersonId;
+use MatchBot\Domain\Salesforce18Id;
+use MatchBot\Domain\StripeCustomerId;
+use MatchBot\Tests\TestCase;
+use Ramsey\Uuid\Uuid;
+
+class DoctrineDonationRepositoryTest extends IntegrationTest
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    public function testFindDonationsToSetPaymentIntent(): void
```

Comment:
I think this test can be made a bit clearer - possibly split into 2 or three tests. Might be possible to rename it to express some more of what we're testing beyond what function we're testing.